model_cards/stas/fsmt-wmt19-de-en/README.md

```diff
@@ -0,0 +1,92 @@
+
+---
+
+<!-- This file has been auto-generated by src/transformers/convert_fsmt_original_pytorch_checkpoint_to_pytorch.py - DO NOT EDIT or your changes will be lost -->
```

Comment:
no they wont unless I run `src/transformers/convert_fsmt_original_pytorch_checkpoint_to_pytorch.py` !

Reply:
Sorry, I'm not sure what you are saying, @sshleifer. What seems to be the problem?

Reply:
it's fine to leave.